get-started/index.md

```diff
@@ -59,10 +62,320 @@ Click "Install" and wait for the installation to complete.
 Once the installation is complete, click "Create Cryostat" to create a Cryostat Custom Resource instance. This provides configuration information for the Operator to know
 the specifics of how to deploy your Cryostat instance. Continue to [Setup](#setup).
 
-Note: Alternative methods for installing the operator are described in [Alternate Installation Options](/alternate-installation-options) (not recommended).
+**Note**: Alternative methods for installing the operator are described in [Alternate Installation Options](/alternate-installation-options) (not recommended).
 ## [Setup](#setup)
 
-### Deploying Cryostat
+### [Configuring Applications](#configuring-applications)
+The following sections will briefly describe how to configure your Java applications so that Cryostat is able to discover and monitor them.
```

Comment:
Worthwhile listing out the 3 methods in this paragraph? 

"There are three methods of configuring your Java applications so that Cryostat is able to discover and monitor them. This includes using the Cryostat Agent, using Java Management Extensions (JMX) and a hybrid approach of using the Cryostat Agent with JMX. The following sections will briefly explain how to configure your Java applications."